### DISCUSSION TITLE
Welcome to Skia4Delphi Discussions!

### DISCUSSION THREAD

**Discussion by viniciusfbb** [skia4delphi/skia4delphi#131] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by War3Evo** [comment]
Hi, I am a hobby programmer worked on many open source projects since the 90s, on and off.  I place myself between beginner and expert.  Self-Taught programming since the Atari 130 XE when I was 5 years old.  You'd think I'd be a expert by now, but that's not what happened.

I've developed many skills over the years.  I am a song writer / musician.  I used Sonar Producer as my DAW for music production.  I took school for wielding did 2 years.  I learned Stick, Tig, and Mig.... then I never used that knowledge for anything.  I have a 2 years diploma in Industrial Systems Technology and used it only for 5 years. I have 7 years as a Trimble and Topcon surveyor for solar construction.

Right now, I'm unemployed.  I lost my driver's license till December.  So, until then... I'm just going to program my time away as I wait for my driver's license.. Then I'll be able to work again.
